specs/data_structures.md

```diff
@@ -322,29 +285,111 @@ A proof into an SMT is structured as:
 
 
 
+## Share
+
+| name      | type               | description     |
+| --------- | ------------------ | --------------- |
+| `rawData` | `byte[SHARE_SIZE]` | Raw share data. |
+
+# Available Data
+
 ## TransactionData
 
-Wrapper for transaction data, which is a simple list of [Transaction](#transaction)s.
+| name                  | type                                          | description                   |
+| --------------------- | --------------------------------------------- | ----------------------------- |
+| `wrappedTransactions` | [WrappedTransaction](#wrappedtransaction)`[]` | List of wrapped transactions. |
 
- | name           | type                            | description           |
- | -------------- | ------------------------------- | --------------------- |
- | `transactions` | [Transaction](#transaction)`[]` | List of transactions. |
+### WrappedTransaction
 
-TODO define a Transaction format
+| name           | type          | description                                                                                     |
+| -------------- | ------------- | ----------------------------------------------------------------------------------------------- |
+| `index`        | `uint64`      | Index of this transaction in the list of wrapped transactions. This is needed for fraud proofs. |
+| `transaction`  | `Transaction` | Actual transaction.                                                                             |
+| `messageStart` | `uint64`      | _Optional_. Starting share of message this transaction pays for.                                |
```

Comment:
Is that the share index?

Reply:
It should be the share index. In fact, with the "tiling" scheme, messages always start at the start of a share.

Reply:
Fixed in 71afd1f.